Data-Storage/material_conversions.json

```diff
@@ -522,7 +527,7 @@
   {
     "id": 257,
     "type": 0,
-    "name": "iron_shovel"
+    "name": "iron_pickaxe"
```

Comment:
really?

Reply:
Yeahhhh... I'm gonna double check the rest when I get home. That one stood out cause it had forgotten pickaxe as the default spear :p